bin/start_be.sh

```diff
@@ -308,6 +308,11 @@ export LIBHDFS_OPTS="${final_java_opt}"
 # see https://github.com/apache/doris/blob/master/docs/zh-CN/community/developer-guide/debug-tool.md#jemalloc-heap-profile
 export JEMALLOC_CONF="percpu_arena:percpu,background_thread:true,metadata_thp:auto,muzzy_decay_ms:30000,dirty_decay_ms:30000,oversize_threshold:0,lg_tcache_max:16,prof_prefix:jeprof.out"
 
+# We can use env vars and not need /etc/krb5.conf when KRB5_CONFIG value in env
+if [[ ${KRB5_CONFIG} ]]; then
```

Comment:
No need.
You can just specify it in be.conf

Reply:
ok.  Deleted.